.github/workflows/update-specs.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Send PRs to the subscribed ECS Agents if the spec files (JSON) are modified
```

Comment:
Since we use "pull" model instead of the previously used "push" we can adapt this comment to something like
```suggestion
# Open a PR if the shared spec files are modified
```